view/adminhtml/web/vue/menu-type.vue

```diff
@@ -47,23 +44,47 @@ define(["Vue"], function(Vue) {
         props: ['item', 'config'],
         data: function() {
             return {
-                draft: {}
+                draft: {},
+            }
+        },
+        computed: {
+            options: function() {
+                var list = [];
+                for (type in this.config.nodeTypes) {
```

Comment:
You can do it simpler

```js
return this.config.nodeTypes.map(type => {
  return {
    label: this.config.nodeTypes[type],
    value: type
  }
})

Reply:
but I'm not using bible to use map and arrow function